### ISSUE TITLE
Weird Imports

### ISSUE THREAD

**Issue by Cabalist** [biologyguy/BuddySuite#65]
The imports seem really weird to me.  There is a try/except clause and editing of the sys.path.  Removing all of that and just using standard imports works for me on both linux and mac.   Is there a reason for this?  

i.e.
```python
try:
    from buddysuite import buddy_resources as br
    from buddysuite import SeqBuddy as Sb
    from buddysuite import MyFuncs
except ImportError:
    import buddy_resources as br
    import SeqBuddy as Sb
    import MyFuncs
```

becomes:

```python
from buddysuite import buddy_resources as br
from buddysuite import SeqBuddy as Sb
from buddysuite import MyFuncs
```

**Comment by biologyguy** [comment]
These are hacks that allow our tests to run from within a Docker image. It's not ideal, but it's currently allowing us to focus on more pressing matters.

**Comment by biologyguy** [comment]
I've finally sorted out relative imports! No more hacking sys.path or catching import errors.
Closed with commit 6e4d2783e8091c52b667dd5b178f4d6d04fa1bd8
